test/conformance/helpers/sources/source_crd_rbac_test_helper.go

```diff
@@ -18,27 +18,27 @@ package sources
 
 import (
 	"context"
+	"strings"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
 	testlib "knative.dev/eventing/test/lib"
 	"knative.dev/pkg/apis/duck"
 
 	rbacv1 "k8s.io/api/rbac/v1"
 )
 
-var clusterRoleName = "eventing-sources-source-observer"
+var clusterRoleNameSuffix = "source-observer"
 var clusterRoleLabel = map[string]string{
 	duck.SourceDuckVersionLabel: "true",
 }
 
 /*
 	The test checks for the following in this order:
 	1. Find cluster roles that match these criteria -
-		a. Has name "eventing-sources-source-observer",
+		a. Has suffix "source-observer" in name like "foos-source-observer",
```

Comment:
the name of the role is not in the spec, so we don't need to test for this.

Reply:
Ok, makes sense. Removed that requirement in latest commit eaa0d2a